ocaml/testsuite/tests/typing-layouts-float64/basics.ml

```diff
@@ -222,20 +222,12 @@ Error: Type t_float64 has layout float64.
    constructor args. *)
 type t5_4 = A of t_float64;;
 [%%expect{|
-Line 1, characters 12-26:
-1 | type t5_4 = A of t_float64;;
-                ^^^^^^^^^^^^^^
-Error: The enabled layouts extension does not allow for mixed constructors.
-       You must enable -extension layouts_beta to use this feature.
+type t5_4 = A of t_float64
```

Comment:
I think the comment just above here is now stale.  And another slightly out of date comment on line 561 way below.